### DISCUSSION TITLE
Webhooks > Subscription v Subscriptions - eventTypes Array contain multiple values?

### DISCUSSION THREAD

**Discussion by OnWednesdaysWeWearFlannel** [freightapis/ssc#9] [Q&A]
Should the eventTypes array should contain only one value or multiple values? The descriptions for the endpoints say "The event types SHOULD include one of two types" but the examples given show the field as an array which implies that it should take multiple values, the examples show it containing multiple values, and the "Responses" section describes the array as an "Array of strings" that contains "The type of events that will be sent to the registered callback."

**Comment by miller79** [comment]
I believe you are referring to the Webhooks section. Looking through the documentation, it needs a bit of adjustment as it implies that it expects only one option. However, the API does allow for multiple values for `eventTypes`. I will bring this back to the team to see if we can update the documentation to be more readable here.

**Reply by OnWednesdaysWeWearFlannel** [reply]
Yes we are referring to the webhooks sections of both subscription and subscriptions. Please let us know when the updates are complete so we can review and see if we still have questions. Thank you!

**Reply by miller79** [reply]
I have created a PR with the fix [here](https://github.com/freightapis/ssc/pull/16). If you have any comments please share and we will see if we can get this pushed through. Thanks again for bringing this up.
